manifests/m/Microsoft/WindowsAppRuntime/1/1/1.1.5/Microsoft.WindowsAppRuntime.1.1.installer.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=AUSU.CRLF.5-1-19041-2364.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.WindowsAppRuntime.1.1
+PackageVersion: 1.1.5
+InstallerType: exe
+InstallModes:
+- silentWithProgress
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- InstallerType: msix
+- DisplayVersion: 1005.616.1651.0
```

Comment:
```suggestion
AppsAndFeaturesEntries:
- InstallerType: msix
  DisplayVersion: 1005.616.1651.0
```

Reply:
Whoops; Was going to quick to notice, good catch!